File: 프로그래머스/unrated/181879. 길이에 따른 연산/길이에 따른 연산.java
class Solution {
    public int solution(int[] num_list) {
        int temp = 1;
        if(num_list.length >= 11) {
            for(int item : num_list) {
                temp += item;
            }
            return temp - 1;
        }else if(num_list.length <= 10) {
            for(int item : num_list) {
                temp *= item;
            }
            return temp;
        }
        return temp;
    }
}

File: 프로그래머스/unrated/181879. 길이에 따른 연산/길이에 따른 연산.js
function solution(num_list) {
    if(num_list.length >= 11) {
        let result = num_list.reduce((acc, curr) => acc += curr );
        return result
    }else if(num_list.length <= 10) {
        let result = num_list.reduce((acc, curr) => acc *= curr );
        return result
    }
}